gradio/blocks.py

```diff
@@ -748,6 +750,11 @@ def launch(
         ssl_certfile: Optional[str] = None,
         ssl_keyfile_password: Optional[str] = None,
         quiet: bool = False,
```

Comment:
I'm afraid that adding all these configurables to the launch method will lead to expanding our argument list endlessly. Here's what I suggest instead: create a method called `configure_queue` to blocks, and add the parameters there. This method will modify the Queue class directly. Two benefits to this approach:
- We don't end up with this ever-expanding argument list to `launch`
- All the queue configuration attributes are grouped together in one spot.

Reply:
I like it!